docs/upgrade.html

```diff
@@ -37,6 +37,8 @@
         A new close API with timeout has been added to <code>KafkaConsumer</code> to control the maximum wait time.</li>
     <li>Multiple regular expressions separated by commas can be passed to MirrorMaker with the new Java consumer via the --whitelist option. This
         makes the behaviour consistent with MirrorMaker when used the old Scala consumer.</li>
+    <li>Zookeeper dependency was removed from the Streams API. Now the Streams API uses Kafka protocol to manage internal topics instead of modifying Zookeeprt directly. This eliminates the need for privilege[s] for accessing Zookeeper directly. If the Kafka cluster is secured the Streams apps require to have proper security privilege[s] to create new topics.</li>
+    <li>Several new fields including SECURITY_PROTOCOL_CONFIG, CONNECTIONS_MAX_IDLE_MS_CONFIG, RETRY_BACKOFF_MS_CONFIG RECONNECT_BACKOFF_MS_CONFIG and REQUEST_TIMEOUT_MS_CONFIG were added to StreamsConfig. User should pay attenntion to the default values and set these if needed.</li>
```

Comment:
We need to provide the actually string name of the config instead of the variable names as they are internal variables and users would not recognize them. Also add a link to the configs section mentioning "please find the detailed description blah blah.."